src/pages/ProfileCreation/ProfileCreation.tsx

```diff
@@ -60,7 +60,7 @@ const ProfileCreation: React.FC = () => {
   };
 
   const saveUsername = async () => {
-    await fetch(process.env.BACKEND_API_URL + "username", {
+    await fetch(process.env.BACKEND_API_URL + "/username", {
```

Comment:
do not change tit here, the pattern here must match the .env.sample example

Reply:
This is so we don't need ending slash in process.env.BACKEND_API_URL